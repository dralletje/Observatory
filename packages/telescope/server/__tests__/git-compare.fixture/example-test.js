it('should match this snapshot', () => {
  expect({
    number: 10,
    null: null,
    array_empty: [],
    array_numbers: [1, 2, 3],
    object_empty: {},
    object: {
      simple_key: 'simple_value',
      "**!!## weird-key $$%%^^": "Weird value 123,.;][';`````']",
    },
    extra_key: null,
  }).toMatchSnapshot();
})
