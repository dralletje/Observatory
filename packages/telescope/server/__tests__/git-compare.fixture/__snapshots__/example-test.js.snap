// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match this snapshot 1`] = `
Object {
  "array_empty": Array [],
  "array_numbers": Array [
    1,
    2,
    3,
  ],
  "extra_key": null,
  "null": null,
  "number": 10,
  "object": Object {
    "**!!## weird-key $$%%^^": "Weird value 123,.;][';\`\`\`\`\`']",
    "simple_key": "simple_value",
  },
  "object_empty": Object {},
}
`;
